src/main/java/htsjdk/tribble/gff/Gff3Codec.java

```diff
@@ -49,6 +47,7 @@
     private static final int FEATURE_TYPE_INDEX = 2;
     private static final int START_LOCATION_INDEX = 3;
     private static final int END_LOCATION_INDEX = 4;
+    private static final int SCORE_INDEX = 5;
```

Comment:
Woops, I probably should have used the power of counting here during the initial review...